docs/variants/protectli_vp2410/hardware-matrix.md

```diff
@@ -0,0 +1,34 @@
+# Hardware configuration matrix - Protectli VP2410
+
+## Introduction
+
+This document describes the hardware configuration used for validation of the
+coreboot port on the Protectli VP2410 firewall.
+
+## Protectli VP2410 Dasharo v1.0.15
+
+| Component              | Description                                              |
+|------------------------|----------------------------------------------------------|
+| **CPU**                | Intel Celeron J4125                                      |
+| **SSD**                | M.2 SSD SATA: Hoodisk SSL032GTTC7-S9A-2S                 |
+|                        | external SATA 2.5 inch: Goodram SSDPR-CL100-240-G2 240GB |
+| **RAM**                | CRUCIAL CT4G4SFS824A                                     |
+|                        | SAMSUNG M471A4G43MB1-CTD                                 |
+|                        | SAMSUNG M471A1K43CB1-CTD                                 |
+|                        | SAMSUNG M471A5244BB0-CRC                                 |
+|                        | KINGSTON KVR24S17S8/8                                    |
+|                        | KINGSTON KVR26S19S8/16                                   |
+| **Wireless card**      | Intel Wi-Fi 6 AX201                                      |
```

Comment:
Currently, we have mounted: `Intel Wi-Fi 6 AX200 (rev 1a)`

Reply:
Fixed